bonsai/sdk/Cargo.toml

```diff
@@ -7,6 +7,14 @@ license = { workspace = true }
 homepage = { workspace = true }
 repository = { workspace = true }
 
+[package.metadata.docs.rs]
```

Comment:
Are you using some editor that automatically formats Cargo.toml files? I'm ok with them being consistently formatted, but I'd like for there to be a CI action to do a check and also a local command line to use to autoformat.